deployment/helm/balloons/README.md

```diff
@@ -0,0 +1,44 @@
+# Introduction
+
+This chart deploys balloons Node Resource Interface (NRI) plugin. The balloons NRI resource
+policy plugin implements workload placement into “balloons” that are disjoint CPU pools.
+
+# Installing the Chart
+
+Path to the chart: `nri-resource-policy-balloons`
+
+```
+helm repo add nri-plugins https://containers.github.io/nri-plugins
+helm install my-balloons nri-plugins/nri-resource-policy-balloons --namespace kube-system
+```
+
+The command above deploys balloons NRI plugin on the Kubernetes cluster within the
+`kube-system` namespace with default configuration. 
+
+# Uninstalling the Chart
+
+To uninstall the balloons plugin run the following command:
+
+```
+helm delete my-balloons --namespace kube-system
+```
+
+# Configuration options
+
+The tables below present an overview of the parameters available for users to customize with their own values,
+along with the default values.
+
+| Name                     | Default                                                                                                                       | Description                                          |
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`             | [ghcr.io/containers/nri-plugins/nri-resource-policy-balloons](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)    | container image name                                 |
```

Comment:
The link should be an URL, like `[ghcr.io/containers/...](https://ghcr.io/containers/...)`